src/Libraries/Microsoft.AspNetCore.Testing/ServiceFakesHostExtensions.cs

```diff
@@ -5,71 +5,26 @@
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
-using System.Net;
 using System.Net.Http;
-using System.Security.Cryptography.X509Certificates;
-using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Hosting.Server;
 using Microsoft.AspNetCore.Hosting.Server.Features;
+using Microsoft.AspNetCore.Testing;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Options;
+using Microsoft.Shared.DiagnosticIds;
 using Microsoft.Shared.Diagnostics;
 
-namespace Microsoft.AspNetCore.Testing;
+namespace Microsoft.Extensions.Hosting;
 
 /// <summary>
 /// Extension methods supporting Kestrel server unit testing scenarios.
 /// </summary>
-public static class ServiceFakesExtensions
+[Experimental(diagnosticId: Experiments.AspNetTesting, UrlFormat = Experiments.UrlFormat)]
```

Comment:
CC: @geeknoid to confirm this.

Reply:
No need for [Experimental] here.

For renames, until November 2023, you can just do the rename and then run scripts/MakeApiBaselines.ps1 to update the manifest file.

After November, this would be a breaking change so then the process will be different.